.github/workflows/build-node-bridge.yml

```diff
@@ -0,0 +1,66 @@
+name: "[Build] suite-node-bridge-bin-js"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    types: [labeled]
```

Comment:
This makes no sense; it will build a node bridge for any PR that you add a label to. Why??

Reply:
Because I do not know workflow syntax well and I copied it like this from some other .yml file. Agreed that it is redundant, I will remove it.

Do you have some other possible improvements, when I will be already making another PR with changes?

Reply:
This was the only thing that i noticed

Reply:
Thanks for that. I created a PR for it - https://github.com/trezor/trezor-suite/pull/14742